doc: Update the supported coverage formats (#48)
codacy/coverage-parser: README.md

Scala library for parsing coverage reports.

The parsers in this project receive as input two parameters, the project root folder and the coverage report.
As a result of the parse operation, it will produce a CoverageReport, a class that is compatible with the
[Codacy coverage format](https://support.codacy.com/hc/en-us/articles/207279819-Coverage).

## Supported formats

> Check the table for the formats we support and which coverage tools write to them:

Reviewer: ```suggestion
Check the table for the formats we support and which coverage tools generate them:
```

Scala library for parsing coverage reports.

> The parsers in this project receive as input two parameters, the project root folder and the coverage report.

Reviewer 2: ```suggestion
The parsers in this project receive as input two parameters, the project root folder, and the coverage report file.
```

> As a result of the parse operation, it will produce a CoverageReport, a class that is compatible with the

Reviewer 2: ```suggestion
As a result of the parse operation, it will produce either a CoverageReport, a class that is compatible with the
```

> [Codacy coverage format](https://support.codacy.com/hc/en-us/articles/207279819-Coverage).

Reviewer 2: ```suggestion
[Codacy coverage format](https://support.codacy.com/hc/en-us/articles/207279819-Coverage) or a string describing the
error parsing the coverage report file.
```

Check the table for the formats we support and which coverage tools generate them:

| Language   | Coverage tools (examples) | Formats   |
| ---        | ---                       | ---       |
| Java       | [JaCoCo](http://eclemma.org/jacoco/) <br> [Cobertura](http://cobertura.github.io/cobertura/) | JaCoCo <br> Cobertura |
| Scala      | [sbt-jacoco](https://www.scala-sbt.org/sbt-jacoco/) <br> [scoverage](http://scoverage.org/) | JaCoCo <br> Cobertura |
| Javascript | [Istanbul](https://github.com/gotwarlost/istanbul) <br> [Poncho](https://github.com/deepsweet/poncho) <br> [Mocha](http://mochajs.org/) + [Blanket.js](https://github.com/alex-seville/blanket) | LCOV |
| Python     | [Coverage.py](https://coverage.readthedocs.io/en/coverage-5.0.3/) | Cobertura                 |
| PHP        | [PHPUnit](https://phpunit.readthedocs.io/en/8.5/code-coverage-analysis.html) | PHPUnit XML <br> [Clover](https://confluence.atlassian.com/clover/using-clover-for-php-420973033.html) |
| Ruby       | [SimpleCov](https://github.com/colszowka/simplecov) | [Cobertura](https://github.com/dashingrocket/simplecov-cobertura) <br> [LCOV](https://github.com/fortissimo1997/simplecov-lcov) |

> Even if you cannot find your language or coverage tool of choice in this table,

Reviewer 2: ```suggestion
You can use this parser with any of the listed coverage formats, even if your language or coverage tool of choice is not in the table above.
```

> you can still use this parser for the supported formats listed above.

Reviewer 2: ```suggestion
```
